ext/hermann/hermann_lib.c

```diff
@@ -359,22 +353,48 @@ static void consumer_consume_stop_callback(void *ptr) {
 }
 #endif
 
+void *consumer_recv_msg(void *ptr)
```

Comment:
Would you mind adding some documentation for doxygen on what this functions purpose is?